assets/js/blocks/cart-checkout/mini-cart-contents/inner-blocks/empty-mini-cart-contents-block/edit.tsx

```diff
@@ -1,40 +1,39 @@
 /**
  * External dependencies
  */
-import { __ } from '@wordpress/i18n';
 import { useBlockProps, InnerBlocks } from '@wordpress/block-editor';
-import { innerBlockAreas } from '@woocommerce/blocks-checkout';
-import type { TemplateArray } from '@wordpress/blocks';
 import { useEditorContext } from '@woocommerce/base-context';
+import { getBlockTypes } from '@wordpress/blocks';
 
 /**
  * Internal dependencies
  */
-import { useForcedLayout, getAllowedBlocks } from '../../../shared';
 
-export const Edit = ( { clientId }: { clientId: string } ): JSX.Element => {
+const excludedBlocks = [
+	'woocommerce/mini-cart',
+	'core/template-part',
+	'core/post-template',
+	'core/comment-template',
+];
```

Comment:
My suggestion here is to cast this `as const`. This will let TypeScript know that this is a readonly array.

Also, perhaps, worth changing this to CONSTANT_CASE

Reply:
Great point! I changed this to a read-only array of strings.